pipeline.yaml

```diff
@@ -39,11 +39,15 @@ data_scoring:
 # Specifies the name of the column containing targets / labels for model training and evaluation
 target_col: "fare_amount"
 steps:
+  ingest:
+    # Skip dataset profiling. Step card will only show run summary information.
+    skip_data_profiling: False
```

Comment:
I think ingest profiling is actually useful. How about turning off data profiling in split and transform by default? i.e. setting them to True in these 2 steps. Also let's discuss this in Slack to reach consensus before merging.